readme.md

```diff
@@ -173,6 +174,59 @@ await adb.Document.PutDocumentAsync(
     item);
 ```
 
+#### Example using graphs
+
+```csharp
+            //Example using graphs, vertices and edges
+            string graphName = "SchoolGraph";
+            string fromClx = "Adults";
+            string toClx = "Students";
+            string edgeClx = "Parents";
+
+            // Create a new graph
+            await adb.Graph.PostGraphAsync(new PostGraphBody
+            {
+                Name = graphName,
+                EdgeDefinitions = new List<EdgeDefinition>
+                {
+                    new EdgeDefinition
+                    {
+                        From = new string[] { fromClx },
+                        To = new string[] { toClx },
+                        Collection = edgeClx
+                    }
+                }
+            });
+
+            // Create a document in the Adults vertex collection
+            PostDocumentResponse<object> fromResponse = await
+                adb.Document.PostDocumentAsync<object>(
+                fromClx,
+                new { Name = "John Doe" });
+
+            // Create a document in the Students vertex collection
+            PostDocumentResponse<object> toResponse = await
+                adb.Document.PostDocumentAsync<object>(
+                toClx,
+                new { Name = "Jimmy Doe" });
+
+            // Create the edge Parent edge between the Adult (John Doe) and Child (Jimmy Doe)
+            var response = await adb.Graph.PostEdgeAsync(
+                graphName,
+                edgeClx,
+                new
+                { 
+                    _from = fromResponse._id,
+                    _to = toResponse._id,
+                    myKey = "parent1"
+                },
+                new PostEdgeQuery
+                {
+                    ReturnNew = true,
+                    WaitForSync = true
```

Comment:
Just by curiosity: Is this considered a good practice to wait for sync when adding edges? Or was it mostly for the example?

Reply:
That's only for the example.